nodejs/spa/src/main/webapp/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
It's important to provide a charset to the document to avoid unexpected behavior.
```suggestion
<head>
    <meta charset="utf-8">
```